src/Components/Checkbox.tsx

```diff
@@ -20,10 +20,32 @@ const CheckboxItem = styled.li`
   }
 
   input {
```

Comment:
Can you add focus styles here? You can also use the `:checked` pseudo class  selector here if that's simpler than passing props.